downloader/src/main/kotlin/vcs/Mercurial.kt

```diff
@@ -23,7 +23,7 @@ import com.here.ort.downloader.VersionControlSystem
 import com.here.ort.downloader.WorkingTree
 import com.here.ort.model.VcsInfo
 import com.here.ort.model.VcsType
-import com.here.ort.spdx.LicenseFileMatcher
+import com.here.ort.utils.LicenseFileMatcher
```

Comment:
That entry should be below `CommandLineTool`.

Reply:
Done.